### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AtillaTahak**
**Hi 👋 Code Reviewer,**

We (@softwareengineermwi , @Munsa1, and I ) have successfully completed all tasks in this assignment.
In a nutshell:

We used the API endpoints to fetch rockets and missions, dragons.

We dispatched the data in redux using thunk middleware.

Added extra data to the data in our Redux store to make a reservation app.

Figured out how to merge our separate contributions into 1 whole program.

This is our deployment link: https://lucid-kalam-d798af.netlify.app/Dragons

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Math Magicians
+
+> React based single page app.  
+
```
Comment (AshakaE) [inline README.md]: - [ ] Kindly improve the description of the project, it's a bit short

File: README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# Math Magicians
```
Comment (AshakaE) [inline README.md]: - [ ] Please update the title of the project

File: src/App.test.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```
Comment (AshakaE) [inline src/App.test.js]: - [ ] Please update this test case it's not related to your project

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi Team 🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights ⚡ 
Your:
   - Project is impressive 👏 
   - Git flow ✔️ 
   - Pull request 👍 

#  Required Changes ♻️ 🚧
_Check the comments under the review._

- [ ] Please write **unit tests** for your project 🙏 

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] 

You can also consider:
- Adding an option to read more about reservations or cancel a reservation from the profile page 💡 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi team, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
### Please go ahead and merge your branch!
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**